packages/react/tests/e2e-storybook/cypress/integration/LinkList/LinkList.e2e.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Sets the correct path
+ *
+ * @type {string}
+ * @private
+ */
+const _pathEndOfSection =
+  '/iframe.html?id=components-link-list--end-of-section';
+
+const getCssPropertyForRule = (rule, prop, sheets) => {
```

Comment:
This is great, I wonder if we can pull this out into a util folder or something so other tests can use this

Reply:
ooh yeah makes sense. i'll change that